src/builtin/module_builtin_ast_serialize.cpp

```diff
@@ -764,6 +784,21 @@ namespace das {
         Expression::serialize(ser);
     }
 
+    AstSerializer & AstSerializer::operator << ( BasicStructureAnnotation::StructureField & field ) {
+        *this << field.name << field.cppName << field.aotPrefix << field.aotPostfix;
+        return *this << field.decl << field.constDecl << field.offset;
+    }
+
+    void BasicStructureAnnotation::serialize ( AstSerializer & ser ) {
+        ser << fields << fieldsInOrder;
+        // ser << helpA;
+        // ser << sti;
+        // ser << mlib;
+        ser << parents;
+        ser << validationNeverFails;
+        // ser << walkMutex;  // Is this always unlocked during serialization?
```

Comment:
Just to make sure: walkMutex is always unlocked during the serialization process and I also need to serialize DebugInfoHelper.